hack/test.sh

```diff
@@ -17,4 +17,4 @@ set -e
 
 echo "> Test"
 
-GO111MODULE=on ginkgo -mod=vendor $@
+GO111MODULE=on go test -race -mod=vendor $@
```

Comment:
What about `--skipPackages`? See https://github.com/gardener/gardener-extension-provider-aws/blob/master/Makefile#L131
Is the recommended way to not enlist the `./test/...` argument anymore now?

Reply:
👍 sounds good